ajax/libs/timelinejs/package.json

```diff
@@ -1,28 +1,15 @@
 {
   "name": "timelinejs",
   "filename": "js/timeline-min.js",
-  "version": "2.36.0",
+  "version": "3.4.7",
   "description": "TimelineJS is an open-source tool that enables you to build visually-rich interactive timelines and is available in 40 languages.",
   "homepage": "https://timeline.knightlab.com/",
   "keywords": [
     "timeline"
   ],
-  "autoupdate": {
```

Comment:
Here, I think you should not delete or change the original auto-update config until you want to update it.